guide/ops/brooklyn_properties.md

```diff
@@ -211,23 +211,10 @@ or
 {% include java_link.html class_name="EntitlementManager" package_path="org/apache/brooklyn/api/mgmt/entitlement" project_subpath="api" %}.
 
 
-## HTTPS Configuration
 
-To enable https, you will need a server certificate in a java keystore. To create a self-signed certificate, you can use the
-following command:
+## Configuring HTTPS in Brooklyn
```

Comment:
I think you need to change `section: HTTPS Configuration` at the top too if you're going to change the titles.